apps/backoffice-v2/src/lib/blocks/hooks/useUbosUserProvidedBlock/useUbosUserProvidedBlock.tsx

```diff
@@ -64,7 +64,7 @@ export const useUbosUserProvidedBlock = ubosUserProvided => {
                   ...rest
                 }) => ({
                   ...rest,
-                  name: `${firstName} ${lastName}`,
+                  name: `${firstName ?? ''}${lastName ? ` ${lastName}` : ''}`,
```

Comment:
This might work? It's a bit shorter and simpler.
```suggestion
                  name: `${firstName || ''}${lastName || ''}`,
```